### PR TITLE
RATIS-1800. Remove unused ratis-shell properties

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
## What changes were proposed in this pull request?
Remove unused properties ratis.shell.user.logs.dir

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/RATIS-1800

## How was this patch tested?

Not need

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
ping @codings-dan

**Review by codings-dan — APPROVED** [review]
the change looks good, +1

**Comment by AngersZhuuuu** [comment]
@codings-dan Seems the failed GA is not related?

File: ratis-shell/src/main/libexec/ratis-shell-config.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ if [[ -n "${RATIS_SHELL_HOME}" ]]; then
   RATIS_SHELL_JAVA_OPTS+=" -Dratis.shell.home=${RATIS_SHELL_HOME}"
 fi
 
-RATIS_SHELL_JAVA_OPTS+=" -Dratis.shell.conf.dir=${RATIS_SHELL_CONF_DIR} -Dratis.shell.logs.dir=${RATIS_SHELL_LOGS_DIR} -Dratis.shell.user.logs.dir=${RATIS_SHELL_USER_LOGS_DIR}"
+RATIS_SHELL_JAVA_OPTS+=" -Dratis.shell.conf.dir=${RATIS_SHELL_CONF_DIR} -Dratis.shell.logs.dir=${RATIS_SHELL_LOGS_DIR}"
```
[the remark this answers is not in the document]
Comment (AngersZhuuuu) [reply ratis-shell/src/main/libexec/ratis-shell-config.sh]: Yea, not need too, also I remove 
```
-if [[ -n "${RATIS_SHELL_HOME}" ]]; then
  RATIS_SHELL_JAVA_OPTS+=" -Dratis.shell.home=${RATIS_SHELL_HOME}"
fi
```

By the way, RATIS_SHELL_ASSEMBLY_CLIENT_JAR really necessary?

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Pull request merged**